### PR TITLE
fixes on create user in service and encryption of password

### PR DESCRIPTION
**Pull request by mathildegrimal**
#### Before submitting, make sure:
- [x] You have rebased your branch
- [x] You describe your changes
- [] You describe How to test
- [ ] I work on the same file as one of my colleague which one?

#### Describe below your changes:
create user function fixed and upgraded with encryption of user's password

### CODE REVIEW

File: src/users/users.service.ts [hunk 1]
```diff
@@ -23,13 +23,22 @@ export class UsersService {
...
+      password: await this.encryptPassword(password),
+      email,
+    });
+
+    //TODO : check password confirm
+    // if (password !== passwordConfirm) {
+    //   throw new PasswordNotMatching();
+    // }
+
+    const payload = await this.usersRepository.save(newUser);
```
Comment (Antonin-dev) [inline src/users/users.service.ts:38]: can you use restructuring method for this 
const { email} = await ......

File: src/users/users.service.ts [hunk 2]
```diff
@@ -23,13 +23,22 @@ export class UsersService {
...
+
+    Logger.log('User created with email ' + payload.email, 'UserModule');
+    return { email: payload.email };
```
Comment (Antonin-dev) [inline src/users/users.service.ts:41]: return {email} only

**Review by Antonin-dev — APPROVED** [review]

File: src/users/users.service.ts (hunk 1 above)
Comment (mathildegrimal) [reply src/users/users.service.ts:38]: ok

File: src/users/users.service.ts (hunk 2 above)
Comment (mathildegrimal) [reply src/users/users.service.ts:41]: ok

File: src/users/users.service.ts [hunk 3]
```diff
@@ -23,13 +23,22 @@ export class UsersService {
...
+    });
+
+    //TODO : check password confirm
+    // if (password !== passwordConfirm) {
+    //   throw new PasswordNotMatching();
+    // }
+
+    const payload = await this.usersRepository.save(newUser);
+
+    Logger.log('User created with email ' + payload.email, 'UserModule');
```
[the remark this answers is not in the document]
Comment (mathildegrimal) [reply src/users/users.service.ts:40]: ok

[the remark this answers is not in the document]
Comment (mathildegrimal) [reply src/users/users.service.ts]: changes ok

File: src/users/users.service.ts (hunk 1 above)
Comment (mathildegrimal) [reply src/users/users.service.ts:38]: impossible to destructurate here

File: src/users/users.service.ts (hunk 2 above)
Comment (mathildegrimal) [reply src/users/users.service.ts:41]: idem

**Pull request merged**